08-rails/05-routes-controllers-and-views.md

```diff
@@ -42,3 +47,64 @@ Let's break down this code:
   - **controller and action**: defining the controller and action, split by the `#`. `"users#index"` in this case this would point to the `index` method in the `UsersController` class.
 
 ![Rails Request Cycle](images/rails-request-cycle.jpg)
+
+## Setting Up a Controller
+Before we go further lets set up our own Controller class.  A Rails Controller is the central manager of a Rails application.  It takes requests from the web server, pulls information from data models and makes decisions.  It then hands information to the views for rendering the data back to the user.  Below we will setup a Controller to list a set of book titles.
+
+Rails can go ahead and create a Controller for us with the following commands.
+
+```bash
+bin/rails generate controller Books index
+```
+
+This command has Rails generate a Controller and Views for the /books path, so when you bring up http://localhost:3000/index you will get the following HTML file.  
+
+```html
+<!DOCTYPE html>
```

Comment:
I think this file info might just be more confusing in here - maybe just describe what the page looks like?